notebooks/image_models/solutions/vertex_tuning_image_classification.ipynb

```diff
@@ -0,0 +1,1226 @@
+{
```

Comment:
<p>* I'd remove the reference to <code>creating_tfrecords_at_scale.ipynb</code> since you have the code to populate the bucket</p><p>* What about moving the data setup cell before trying to access the data?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/225/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Agreed, the current flow of the lab is a bit non-linear right now. </p>